src/app/features/date-range/models/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './unavailable-dates';
+export * from './date-range-data';
```

Comment:
it is recommended to hav en empty line at the end of each file.

Reply:
done